sphinxcontrib/drawio.py

```diff
@@ -139,6 +140,9 @@ def render_drawio(self: SphinxTranslator, node: DrawIONode, in_filename: str,
             value = node["config"][option]
             extra_args.append("--{}".format(option))
             extra_args.append(str(value))
+        elif option == "scale":
```

Comment:
Whilst this does what you want it to do, I do not think placing it here conveys exactly what it should do.
If you have a default case, do not use optional_uniques, do it like I have done `page-index`.